index.d.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * Sets the app ID where you'll be sending data. It can be used to switch between projects or
+ * between your production and development environments.
+ *
+ * `setAppId` initializes Heap tracking on iOS. It should be called only once per app open.
+ * If it is called multiple times, it will initiate a new session for the user.
```

Comment:
If it's called more than once in a react native iOS app, subsequent calls will be ignored. @GrandPoohBear is that the behavior on android or will it create a new session?

Reply:
Yeah, unfortunately this behavior is different across platforms right now, but it's something we will have to address from the native lib side.  Current behavior as of 0.1.0:
iOS - Subsequent calls will be ignored, so only the first appId will ever be used (by default, the native lib stops sending events if this gets called twice, so the bridge prevents it.)
Android - This initializes an entirely new tracker object, so the session will be created from scratch.

Reply:
I updated it with a note about the differences between behavior on ios vs android.